DQM/DTMonitorClient/src/DTOccupancyTest.cc

```diff
@@ -283,7 +300,7 @@ int DTOccupancyTest::runOccupancyTest(TH2F *histo, const DTChamberId& chId,
   bool failCells = false;
 
   // Check that the chamber has digis
-  if(histo->Integral() == 0) {
+  if (getIntegral(histo,chId) == 0) {
```

Comment:
probably a check on the maximum would be more efficient here.
referring to TH1::GetMaximum
https://root.cern.ch/doc/master/classTH1.html#acb53c99a65ab29a045cbdc2789e55250

the chId in the list of arguments passed to the function is not needed.